README.md

```diff
@@ -176,7 +176,79 @@ The `tools` object is an array of `Tool` objects. `Tool` objects represent tools
 
 #### Preferences
 
-Last, but not the least, is the `preferences` object which is an array of `PreferencesTool.Section` objects. `PreferencesTool.Section` is used to allow the user to create sections of toggleable options. Each section consists of `ToggleToolItem` objects. To allow the user the interaction, you will have to add a `getter`, and a `setter` for the property from the `UserDefaults` object.
+Last, but not the least, is the `preferences` object which is an array of `PreferencesTool.Section` objects. `PreferencesTool.Section` is used to allow the user to create sections of editable options. Each section consists of `PreferenceItem` objects.
```

Comment:
I'm fine with this, but `README.md` is getting bigger and bigger. I would put this here for now, but if the file will grow a bit more, in future, I would recommend creating a file under `docs` folder and link it in the `README.md` file.

Reply:
I agree, we can create multiple docs files, and divide the whole Readme file 👌